### ISSUE TITLE
IE 11 - [nuxt.js] Cannot load components SyntaxError: Expected ':'

### ISSUE THREAD

**Issue by muthukumarsp** [nuxt/nuxt.js#584]
Hi,
I am getting the following error in IE-11 and JavaScript crashes after the error.  Any help?
I am using child routing using <nuxt-child>.
There is no issue in Chrome and FF.

![ie-11-issue](https://cloud.githubusercontent.com/assets/5599346/25158775/44ac8b82-24ee-11e7-90be-24801e0a9a6e.png)

**Comment by pi0** [comment]
@muthukumarsp you may have to add `babel-polyfill` in your build vendor to fully support IE. (Things like Symbols are not available with transforms and need runtime polyfill)

nuxt.config.js
```js
build: {
        vendor: ['babel-polyfill', ...],
}
```

```bash
yarn add babel-polyfill
```

**Comment by muthukumarsp** [comment]
@pi0 
Thanks for your reply.

I already had a 'babel-polyfill' as follows.
  entry: {
     app: [
      'babel-polyfill'
    ]
  }

I tried adding as per your comment but still I am getting the same issue for this particular child route.

I noticed there is a network request as follows when I navigate to this route, but it fails with '4040' not found.

   Request URL: http://localhost:3000/trading/markets/es6-promise.map
